ata_db_models/models.py

```diff
@@ -66,3 +73,13 @@ class Prescription(SQLModel, table=True):
     last_updated: datetime
     # TODO will add this once we have models to work with!
     # model_id: UUID = Field(foreign_key="model.id")
+
+
+def default_group() -> Group:
+    return random.choice([g for g in Group])
```

Comment:
⛏ Can also do `random.choice([*Group])` haha

Reply:
tried it and it works, but mypy doesn't like it